pq-crypto/bike_r3/aes.h

```diff
@@ -7,7 +7,11 @@
 
 #pragma once
 
-#include <openssl/evp.h>
+#if defined(STANDALONE_IMPL)
```

Comment:
`STANDALONE_IMPL` doesn't seem to be defined anywhere (neither in this PR nor in s2n already). `#if defined(STANDALONE_IMPL)` will always evaluate to false.